release-checks.sh

```diff
@@ -52,6 +52,12 @@ then
     exit 1
```

Comment:
In the line above, we still mention `RELEASE.md` but it is gone by now. Can you please remove it as well?